### PR TITLE
Update npm doc about container insights metrics

### PR DESCRIPTION
**Pull request by vishiy**

### CODE REVIEW

File: articles/aks/monitor-and-visualize-with-NPM.md [hunk 1]
```diff
@@ -60,7 +73,7 @@ InsightsMetrics
 ### Viewing in Grafana Dashboard
 Set up your Grafana Server and configure a Log Analytics Data Source as described [here](CI-grafana-setup). Then, import our [Grafana Dashboard with a Log Analytics backend](CI-grafana-dashboard) into your Grafana Labs.
```
Comment (aanandr) [inline articles/aks/monitor-and-visualize-with-NPM.md:74]: Can you check the link for the Log Analytics Data Source? The link in Hunter's original doc doesnt work. Can you put the right link?

File: articles/aks/monitor-and-visualize-with-NPM.md [hunk 2]
```diff
@@ -60,7 +73,7 @@ InsightsMetrics
...
 
-The dashboard has very similar visuals compared to the Azure Workbook. 
+The dashboard has visuals very similar to the Azure Workbook. 
 
 ## Quick Setup for Prometheus Server
 Some users may choose to collect metrics with a Prometheus Server instead of Azure's native Container Insights. You merely need to add two jobs to your scrape config to collect NPM metrics.
```
Comment (aanandr) [inline articles/aks/monitor-and-visualize-with-NPM.md:79]: Multiple comments. This section needs to be simplified and i need your help
1. Is it necessary to provide steps to install Prometheus? Its there in the original doc but i feel we should let the user/customer worry about it. I dont accurately know which portions to remove and which to keep. For e.g. is the prometheus server addition step required? Or can we just publish the scrape config? 
2. Another question is whether we can just publish the scrape config file somewhere and out here just provide a link (can we let customer worry about downloading it and installing it?)
3. What is the purpose of the second portion which talks about replacing azure-npm-node-metrics job with an alternate one? Cant we just have one?

**Review by aanandr — CHANGES_REQUESTED** [review]
I have left comments inside. In addition to that i had the following. 

Can you fix the link for "Grafana Dashboard with a Prometheus backend"? Its broken in Hunter's original doc

File: articles/aks/monitor-and-visualize-with-NPM.md (hunk 1 above)
Comment (vishiy) [reply articles/aks/monitor-and-visualize-with-NPM.md:74]: Updated

File: articles/aks/monitor-and-visualize-with-NPM.md (hunk 2 above)
Comment (vishiy) [reply articles/aks/monitor-and-visualize-with-NPM.md:79]: #1 - Better to leave it here, as its all in one place.
#2 - Yes, better to publish scrape config as a yaml file in your npm repo and point to that
#3 - you need 2 jobs, as each is a different scrape target (one for node metrics & one for cluster metrics)

Comment (aanandr) [reply articles/aks/monitor-and-visualize-with-NPM.md]: #1. By keeping the documentation for installation of Prometheus we are taking the responsibility to make sure it works. I had tried it some weeks back and it didnt work for me . I realized that i was not updating a few things that were mandatory. Hence my point. Prometheus installation is not specific to NPM. The scrape file is. Let me know now.
#2. Ok i will do this.
#3. The wording clearly says - "You can also replace the azure-npm-node-metrics job above with the following, or incorporate the following into a pre-existing job for Kubernetes pods". It clearly sounds like this is not mandatory. So i am trying to understand why it is here then.

**Comment by aanandr** [comment]
Looks fine. I will now move this to the actual location where we want to publish it

**Pull request merged**